plugins/inputs/ah_wireless/README.md

```diff
@@ -0,0 +1,4381 @@
+  ifIndex:
+    description: radio interface.
+    type: integer
+    format: int32
+  ssid:
+    description: Client SSID name
+    type: string
+  txPackets:
+    description: TX data frame count.
+    type: integer
+    format: int32
+  txBytes:
+    description: TX data byte count.
+    type: integer
+    format: int32
+  txDrop:
+    description: TX frames are dropped due to max retried.
+    type: integer
+    format: int32
+  slaDrop:
+    description: number of SAL violation traps occurred.
+    type: integer
+    format: int32
+  rxPackets:
+    description: RX data frame count.
+    type: integer
+    format: int32
+  rxBytes:
+    description: RX data byte count.
+    type: integer
+    format: int32
+  rxDrop:
+    description: RX frames dropped due to decrypt, MIC error etc.
+    type: integer
+    format: int32
+  avgSnr:
+    description: Average SNR (dB).
+    type: integer
+    format: int8
+  psTimes:
+    description: Client entered into power save mode times.
+    type: integer
+    format: int32
+  radioScore:
+    description: Radio link score (client SLA score).
+    type: integer
+    format: int8
+  ipNetScore:
+    description: IP network connectivity score.
+    type: integer
+    format: int8
+  appScore:
+    description: Application health score.
+    type: integer
+    format: int8
+  phyMode:
+    description: client radio mode.
+    type: integer
+    format: int8
+  rssi:
+    description: client rssi.
+    type: integer
+    format: int8
+  os:
+    description: client OS name
+    type: string
+  name:
+    description: user name
+    type: string
+  host:  
+    description: host name
+    type: string
+  profName:
+    description: user profile name
+    type: string
+  dhcpIp:
+    description: DHCP server IP.
+    type: integer
+    format: int32
+  gwIp:
+    description: gateway IP.
+    type: integer
+    format: int32
+  dnsIp:
+    description: DNS server IP.
+    type: integer
+    format: int32
+  clientIp:
+    description: Client IP address.
+    type: integer
+    format: int32
+  dhcpTime:
+    type: integer
+    format: int32
+  gwTime:
+    type: integer
+    format: int32
+  dnsTime:
+    type: integer
+    format: int32
+  clientTime:
+    type: integer
+    format: int64
+  rxRateStats:
+    description: rx rate counter
+    type: array
+
+  txRateStats:
+    description: tx rate counter
+    type: array
+
+  txNssUsage:
+    description: TX spatial stream percentage.
+    type: array
+    maxItems: 4
+    items:
+      type: integer
+      format: int8
+  txAirtime:
+    description: station TX airtime percentage.
+
+  rxAirtime:
+    description: station RX airtime percentage.
+
+  bwUsage:
+    description: Bandwidth usages.
+
+  required:
+    keys
+
+## Troubleshooting
+
+On HiveOS, we can check output by executing hiden commands:
+_show report snapshot client
+and
+_show report snapshot interface
+
+## Example Output
+
+```shell
+{
+  'rfStats': [
```

Comment:
This sample output appears based on EXTR serializer format (i.e. data_format = "extr") as it leaves the output plugin, e.g. outputs.http.  I don't think output specific data format should be included in the README of an input plugin.  We should probably use standard influxdb format for your example, given the output format will vary depending on which output plugin this data is destined for.

See input plugin like disk or internal for their Example Output.

Reply:
I will remove it.